project/component/src/events/evaluations/evaluations.js

```diff
@@ -0,0 +1,27 @@
+(function () {
+    const EvaluationsEvents = function () {
+        this.init = function (build) {
+            setEvents('new-evaluation', 'newEvaluation', build);
+            setEvents('logout', 'login', build);
+            const detailsButtons = document.getElementsByClassName('details');
+            Array.prototype.slice.call(detailsButtons).forEach((element) => {
+                const id = element.getAttribute('id');
+                setEvents(id, 'details', build);
+            });
+        };
+        this.destroy = function (build) {
+            removeEvents('new-evaluation', setEvents('new-evaluation', 'newEvaluation', build));
+            removeEvents('logout', setEvents('logout', 'login', build));
+
+            const detailsButtons = document.getElementsByClassName('details');
+            Array.prototype.slice.call(detailsButtons).forEach((element) => {
```

Comment:
Nice! As a side note, you should be able to simply do `Array.prototype.forEach.call(detailsButton, ...)` instead.

Reply:
good to know